### ISSUE TITLE
i686 branches are failing validity check

### ISSUE THREAD

**Issue by jonathonf** [manjaro/pacman-mirrors#122]
`pacman-mirrors=4.7.2-3` is failing on i686 due to the "unknown" **x32-** branches,

```bash
sudo pacman-mirrors -a -U 'https://mirror.netzspielplatz.de/manjaro/packages'
.: ERROR: Invalid setting in `/etc/pacman-mirrors.conf`
     'Branch = x32-unstable'; EXPECTED stable|testing|unstable
```
https://github.com/manjaro/pacman-mirrors/blob/a82bfcfcd111d7a277639fd7acb4226d36f533cf/pacman_mirrors/functions/configFn.py#L101

**Comment by fhdk** [comment]
That was a terrible error.

Sorry for the inconvenience - it has been fixed earlier toady with the build of 4.7.4

**Comment by jonathonf** [comment]
No worries. ;)
